main.js

```diff
@@ -160,4 +160,177 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+
+function countInventoryToSell (televisions) {
+  console.log("SALES", televisions);
+  let totalCount = 0;
+
+  for (const television of televisions) {
+    console.log("television in loop", television.originalStock, television.sold);
+
+    const toSellOfThisType = television.originalStock - television.sold;
+    console.log("to sell of this type: ", toSellOfThisType);
+    totalCount = totalCount + toSellOfThisType;
+  }
+  console.log("Total count:", totalCount);
+  return totalCount;
+}
+
+const totalCount = countInventoryToSell(inventory);
+
+
+
+const numberToSell = document.getElementById('number-to-sell');
+numberToSell.textContent = totalCount;
+
+console.log(numberToSell);
+
+
+// Opdracht 2a: Gebruik een array-methode om een array te maken met alle tv-type namen.
+
+const tvNames = inventory.map((name) => {
+  return name.brand + " " + name.name;
+});
+console.log(tvNames);
+
+// Opdracht 2b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn.
+
+
+const soldOut = inventory.filter((television) =>{
+  return television.originalStock === television.sold;
+});
+
+// Opdracht 2c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken.
+
+const ambiLight = inventory.filter((hasAmbiLight)=> {
+  return hasAmbiLight.options.ambiLight === true;
+})
```

Comment:
De `=== true` is in feite overbodig. Je zou dit dus zo kunnen schrijven:
```
const ambiLight = inventory.filter((hasAmbiLight)=> {
  return hasAmbiLight.options.ambiLight;
});
```

Of nog korter:
```
const televisionsWithAmbilight = inventory.filter(television => television.options.ambiLight);
```